examples/misc/legends/legend-categorical-chart.html

```diff
@@ -0,0 +1,162 @@
+<html>
+
+<head>
+  <!-- Include CARTO VL JS -->
+  <script src="../../../../dist/carto-vl.js"></script>
+  <!-- Include Chart JS -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.2/Chart.min.js"></script>
+  <!-- Include Mapbox GL JS -->
+  <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+  <!-- Include Custom CSS -->
+  <link rel="stylesheet" type="text/css" href="../../style.css">
+</head>
+
+<body>
+  <!-- Map goes here -->
+  <div id="map"></div>
+  <aside class="toolbox chart-toolbox">
+    <div class="box chart-box">
+      <header>
+        <h1>Daytime sales</h1>
+      </header>
+      <section>
+        <div id="controls">
+          <canvas id="histogram" width="500" height="600"></canvas>
+        </div>
+      </section>
+      <footer class="js-footer"></footer>
+    </div>
+  </aside>
+  <div id="loader">
+    <div class="CDB-LoaderIcon CDB-LoaderIcon--big">
+      <svg class="CDB-LoaderIcon-spinner" viewBox="0 0 50 50">
+        <circle class="CDB-LoaderIcon-path" cx="25" cy="25" r="20" fill="none"></circle>
+      </svg>
+    </div>
+  </div>
+
+  <script>
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+      center: [5, 40.5],
+      zoom: 5,
+      dragRotate: false,
+    });
+    map.touchZoomRotate.disableRotation();
+
+    const nav = new mapboxgl.NavigationControl({
+      showCompass: false
+    });
+    map.addControl(nav, 'top-left');
+
+    carto.setDefaultAuth({
+      user: 'cartovl',
+      apiKey: 'default_public'
+    });
+
+    const ctx = document.getElementById('histogram').getContext('2d');
+    // Define the source
+    const source = new carto.source.Dataset('ne_10m_populated_places_simple');
+    // Define the visualization
+    const viz = new carto.Viz(
+    ` @color: ramp($featurecla, bold) 
+      @histogram: viewportHistogram($featurecla)
+      color: @color
+      width: 35
+      strokeWidth: 0
+    `
+    );
+
+    // Draw chart using Chart.js library (https://www.chartjs.org/)
+    const chartOptionsDefault = {
+      animation: {
+        duration: 0
+      },
+      legend: {
+        display: false
+      },
+      scales: {
+        yAxes: [{
+          gridLines: {
+            drawBorder: false,
+            display: false
+          },
+          ticks: {
+            suggestedMin: 1,
+            beginAtZero: true,
+            display: false
+          }
+        }],
+        xAxes: [{
+          gridLines: {
+            drawBorder: false,
+            display: false
+          },
+          ticks: {
+            display: false
+          },
+          barPercentage: 1.0,
+          categoryPercentage: 1.0
+        }]
+      }
+    };
+
+    const chart = new Chart(ctx, {
+      type: 'bar',
```

Comment:
I can see some 'trembling' on each update. I'll try to solve it somehow. A debounce like this might work:

```js
    const debounce = function (func, wait = 100) {
      let timeout;
      return function (...args) {
        clearTimeout(timeout);
        timeout = setTimeout(() => {
          func.apply(this, args);
        }, wait);
      };
    }

    const drawHistogramLegend = debounce(_drawHistogramLegend, 100);

    layer.on('loaded', () => {
      hideLoader();
      drawHistogramLegend();
    });

    layer.on('updated', drawHistogramLegend);

    function _drawHistogramLegend() {

      // WHATEVER...
```